### PR TITLE
[fix] Rebase conflict fix for isFusableDevice

### PR DESCRIPTION
**Pull request by bwasti**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#20251 [fix] Rebase conflict fix for isFusableDevice**

Differential Revision: [D15262850](https://our.internmc.facebook.com/intern/diff/D15262850)

### CODE REVIEW

**Review by wanchaol — APPROVED** [review]
looks good if ci pass

File: torch/csrc/jit/passes/graph_fuser.cpp [hunk 1]
```diff
@@ -580,8 +580,7 @@ struct GraphFuser {
     // we can move the consumer up into the producer.
     // but this requires better handling of merging fusion groups so it is not
     // done now
-    bool shouldFuse = isFusableDevice(producer) &&
-        isFusable(producer->node()) &&
+    bool shouldFuse = isFusable(producer->node()) &&
```
Comment (jamesr66a) [inline torch/csrc/jit/passes/graph_fuser.cpp]: ????

**Review by jamesr66a — CHANGES_REQUESTED** [review]
Looks wrong

File: torch/csrc/jit/passes/graph_fuser.cpp (hunk 1 above)
Comment (wanchaol) [reply torch/csrc/jit/passes/graph_fuser.cpp]: this `isFusableDevice` check is handled in Bram's previous PR on custom fusion [here](https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/passes/graph_fuser.cpp#L246)

Comment (jamesr66a) [reply torch/csrc/jit/passes/graph_fuser.cpp]: Okay, so the huge comment block in the middle of a boolean expression was super confusing. Can we move that out? maybe assign that boolean to its own variable then use that in the boolean expression?

Comment (bwasti) [reply torch/csrc/jit/passes/graph_fuser.cpp]: this preserves the change/comment block from here:
https://github.com/pytorch/pytorch/commit/5be4bee4ff58d04c8ee73163e2cb2e06bd69b2a5#diff-52ef1c06d69edbd31a705b6d2b827f50

Comment (bwasti) [reply torch/csrc/jit/passes/graph_fuser.cpp]: basically this diff just fixes a rebase bug

**Comment by bwasti** [comment]
can I land this?

**Pull request closed without merging**